mysql-test/suite/rocksdb/include/locking_issues_case1_1.inc

```diff
@@ -17,11 +17,7 @@
 --echo -   using $isolation_level transaction isolation level
 --echo -----------------------------------------------------------------------
 
---disable_warnings
-DROP TABLE IF EXISTS t0;
---enable_warnings
-
-CREATE TABLE t0(id1 INT, id2 INT, value INT, PRIMARY KEY(id1, id2));
+CREATE TABLE t0(id1 INT, id2 INT, value INT, PRIMARY KEY(id1, id2)) engine=rocksdb;
```

Comment:
Here and everywhere inconsistent SQL syntax case "CREATE" but "engine". I don't know the official RocksDB capitalisation for the plugin SE name, likely it's "RocksDB"